universal-application-tool-0.0.1/project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
 // The Play plugin
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.8")
-addSbtPlugin("com.typesafe.sbt" % "sbt-play-ebean" % "6.0.0")
+addSbtPlugin("com.typesafe.play" % "sbt-play-ebean" % "6.2.0-RC4")
```

Comment:
Is this change required? Both the sbt version change and this are the ones that seem most likely to be associated with the build problem, so doing only one of them would be good for diagnosis.

Reply:
Unfortunately they need to be made together. The newer `play-ebean` versions don't work with sbt 1.3.13 and require at least 1.4y. That being said it may be worth trying a smaller jump for sbt than 1.6.1.